Dockerfile

```diff
@@ -12,12 +12,16 @@ RUN bin/ape.elf bin/python -m compileall Lib
 RUN mv Lib/__pycache__/scrapscript*.pyc Lib/scrapscript.pyc
 RUN rm Lib/scrapscript.py
 RUN cp bin/python bin/scrapscript.com
-RUN sh bin/zip -A -r bin/scrapscript.com Lib .args
+COPY style.css .
+COPY repl.html .
+RUN sh bin/zip -A -r bin/scrapscript.com Lib .args style.css repl.html
 RUN bin/ape.elf bin/assimilate bin/scrapscript.com
 
 # Set up the container
 FROM scratch
 COPY --from=build /cosmo/bin/scrapscript.com .
+COPY style.css .
```

Comment:
I have to figure out how to get rid of one of these. I don't know how to get the right path to the file inside cosmopolitan